pages/common/reflex.md

```diff
@@ -0,0 +1,24 @@
+# reflex
+
+> Reflex is a small tool to watch a directory and rerun a command when certain files change.
+> More information: <https://github.com/cespare/reflex>.
+
+- Rebuild with `make` if any file changes:
+
+`reflex make`
+
+- Compile and run Go application if any .go file changes:
+
+`reflex -r '\.go$' go run .`
+
+- Ignore a directory when wahtcing for changes:
+
+`reflex -R '^node_modules/' yarn test`
+
+- Run command when reflex starts and restarts on file changes:
+
+`reflex -s go test`
```

Comment:
Please use long argument format.

Reply:
@navarroaxel what do you mean by long argument format?

Reply:
```suggestion
`reflex --start-service=false go test`
```

Reply:
@navarroaxel thanks for the clearification. I looked at the style guide, tldr page and some others (zip, ln, watch, entr) and they all favor shorthand arguments over the long arguments.

I don't mind changing my pull request according to your feedback, I just want to make sure it's the right direction.

Reply:
We prefer the use of long arguments if possible, but we have exceptions for them when it's really uncommon the use of long arguments, e.g.  `pacman -Syu`, etc

Reply:
Thanks for making that clear. I updated my changes accordingly.